### PR TITLE
added v basic summary, image, quote

### PR DESCRIPTION
**Pull request by naienko**

### CODE REVIEW

File: main.css [hunk 1]
```diff
@@ -69,4 +69,9 @@ footer ul {
     font-size: 2em;
     color: #A3C6C4;
 }
-/* Hex: 354649 / 6C7A89 / A3C6C4 / E0E7E9 */
\ No newline at end of file
+/* Hex: 354649 / 6C7A89 / A3C6C4 / E0E7E9 */
```
Comment (justinavickers) [inline main.css:75]: example

**Review by justinavickers — DISMISSED** [review]

**Comment by justinavickers** [comment]
👍

**Comment by JanetWoods** [comment]
looks good if you could resolve conflicts first.  Thanks

**Comment by kellycoles** [comment]
👍

**Pull request merged**